inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -142,7 +142,7 @@ if (TBBBIND_2_4_FOUND)
     target_link_libraries(${TARGET_NAME}_obj PRIVATE ${TBBBIND_2_4_IMPORTED_TARGETS})
 endif()
 
-add_cpplint_target(${TARGET_NAME}_cpplint FOR_TARGETS ${TARGET_NAME}_obj)
+add_clang_format_target(${TARGET_NAME}_cpplint FOR_TARGETS ${TARGET_NAME}_obj)
```

Comment:
target name should be different :) 

Can be fixed in a different PR if no more meaningful comments

Reply:
Fixed